fighters/buddy/src/opff.rs

```diff
@@ -23,10 +23,42 @@ unsafe fn grenade_ac(fighter: &mut L2CFighterCommon) {
     }
 }
 
+// Banjo Dair bounce
+unsafe fn dair_bounce(fighter: &mut L2CFighterCommon, boma: &mut BattleObjectModuleAccessor, motion_kind: u64, frame: f32){
```

Comment:
Should really only be importing L2CFighterCommon now, and using fighter exts accordingly

Reply:
Agreed, for characters that we touch who have *really* simple OPFF files, we should clean up the 1-2 existing calls that they have and get rid of superfluous "frameinfo" bs and "moveset" functions when possible.